payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AddressSpec.kt

```diff
@@ -69,24 +71,46 @@ data class AddressSpec(
     fun transform(
         initialValues: Map<IdentifierSpec, String?>,
         addressRepository: AddressRepository
-    ) = createSectionElement(
-        if (displayFields.size == 1 && displayFields.first() == DisplayField.Country) {
-            CountryElement(
-                IdentifierSpec.Generic("billing_details[address][country]"),
-                DropdownFieldController(
-                    CountryConfig(this.allowedCountryCodes),
-                    initialValue = initialValues[this.apiPath]
-                )
+    ): SectionElement {
```

Comment:
Attempted to convert the AddressElement into a `FormElement`, but this became a problem when getting the hidden identifiers. `FormElement` doesn't have a way to access its children like `SectionElement` does. We should refactor the way elements works. Ideally, the Element itself always has access to its children.

Reply:
Here, I am creating a new `SectionElement` with two fields: `AddressElement` and `SameAsShippingElement`